### ISSUE TITLE
Taskbar is cut off when using DPI scaling factor

### ISSUE THREAD

**Issue by lmagder** [zpydr/gnome-shell-extension-taskbar#134]
When setting org.gnome.desktop.interface scaling-factor = 2 the UI scales correctly but the positioning code doesn't seem to take this into account so the bottom half of the taskbar is cut off

![image](https://cloud.githubusercontent.com/assets/2025344/20459830/59fde51a-ae85-11e6-98e9-595e59cd9d5c.png)

**Comment by zpydr** [comment]
You're right, the scaling factor is not taken into account, I will have to implement this. Meanwhile please check the "Anchor Point" setting on the Panels tab. You should be able to correct it this way.

**Comment by lmagder** [comment]
Thanks. I was able to work around it by setting the Anchor Point to be the same value as the taskbar height
